ct/alcopac.sh

```
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+GITHUB_USER="Hlushok"
```

Shellcheck (0.11.0)

[warning] GITHUB_USER appears unused. Verify use (or export if used externally).

(SC2034)